ReactNativeClient/lib/models/Note.js

```diff
@@ -318,6 +324,13 @@ class Note extends BaseItem {
 			}
 		}
 
+		const cond = options.conditions.slice();
+		cond.push('pinned = 1');
+		const tempOptions = Object.assign({}, options);
+		tempOptions.conditions = cond;
```

Comment:
Can this be done nicely without `cond`?

Reply:
Do you want me to rewrite more concisely with the same logic, or change the logic?

Reply:
Removed `cond` from logic